port/unix_include/omrportpg.h

```diff
@@ -120,7 +121,8 @@ typedef struct OMRPortPlatformGlobals {
 #endif
 
 #if defined(LINUX)
-#define PPG_cgroupLimitsEnabled (portLibrary->portGlobals->platformGlobals.cgroupLimitsEnabled)
+#define PPG_cgroupSubsystemsAvailable (portLibrary->portGlobals->platformGlobals.cgroupSubsystemsAvailable)
```

Comment:
Doesn't that state a relationship?  Neither PPG_cgroupSubsystemsAvailable or PPG_cgroupSubsystemsAvailable are valid if PPG_cgroupEntryList is null.  

The code should state that invariant.